pkg/kubelet/config/file.go

```diff
@@ -90,7 +123,7 @@ func (s *sourceFile) resetStoreFromPath() error {
 		}
 		// Emit an update with an empty PodList to allow FileSource to be marked as seen
 		s.updates <- kubetypes.PodUpdate{Pods: []*v1.Pod{}, Op: kubetypes.SET, Source: kubetypes.FileSource}
-		return fmt.Errorf("path does not exist, ignoring")
+		return fmt.Errorf("Path does not exist, ignoring")
```

Comment:
Sorry for the confusion - error messages (i.e. `fmt.Errorf(...)`) should start with a lowercase letter (revert this), but log messages (i.e. `glog.Errorf(...)`) should start with a capital.